frontend/src/modules/schemas/editor/composables/content/useNodeContextMenu.ts

```
@@ -0,0 +1,82 @@
+
+      title: toValue(options.title),
+      actions: toValue(options.actions).filter((action): action is IContextMenuAction => !!action),
+    });
+
+    addEventListener('click', () => plugin.hide(), { once: true, capture: true });
+  });
+}
+
+export function useNodeContextMenu(options: INodeContextMenuOptions): void {
+  const item: INodeContextMenuItem = { ...options, id: newId() };
+  const items = inject(TOKEN)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)